app/views/provider_interface/offer/checks/new.html.erb

```diff
@@ -0,0 +1,80 @@
+<% content_for :browser_title, title_with_error_prefix(t('.title'), @wizard.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(provider_interface_application_choice_new_offer_path(@application_choice.id)) %>
+
+<%= render(FlashMessageComponent.new(flash: flash)) %>
```

Comment:
do we need a flash message here?

Reply:
No, we should not need this in any of the forms as it's rendered in the application layout. I have also removed from the conditions/new page. The old offer pages appear to use it, but as we will be removing them soon I don't think there is any value in revisiting and removing now.